ee/connectors/consumer_pool.py

```diff
@@ -10,7 +10,7 @@ def main():
     database_api = DBConnection(DATABASE)
 
     allowed_projects = config('PROJECT_IDS', default=None, cast=Csv(int))
-    w_pool = WorkerPool(n_workers=60,
+    w_pool = WorkerPool(n_workers=config('OR_EE_CONNECTOR_WORKER_COUNT'),
```

Comment:
I've more of a suggestion regarding naming. I'm more inclined to `CONNECTOR_WORKER_COUNT` than `OR_EE_CONNECTOR_WORKER_COUNT`, coz
1. OR: is obvious
2. EE: app doesn't have the notion of ee or not. It's just business notion.